bin/cron/admin_stats

```diff
@@ -4,47 +4,105 @@
-  teacher_prizes = DASHBOARD_REPORTING_DB_READONLY[:teacher_prizes]
-  teacher_bonus_prizes = DASHBOARD_REPORTING_DB_READONLY[:teacher_bonus_prizes]
 
   Properties.set(:admin_stats, {
-    user_count: users.count,
-    teacher_count: teachers.count,
-    student_count: students.count,
-    users_with_teachers: followers.distinct.count(:student_user_id),
-    users_with_email: users.where('email <> ""').count,
-    users_with_confirmed_email: users.where('confirmed_at IS NOT NULL').count,
-    girls: students.where(:gender => 'f').count,
-    boys: students.where(:gender => 'm').count,
-
-    prizes_redeemed: prizes.where('user_id IS NOT NULL').group(:prize_provider_id).count,
-    prizes_available: prizes.where('user_id IS NULL').group(:prize_provider_id).count,
-
-    student_prizes_earned: users.where(:prize_earned => true).count,
-    student_prizes_redeemed: prizes.where('user_id IS NOT NULL').count,
-    student_prizes_available: prizes.where('user_id IS NULL').count,
-
-    teacher_prizes_earned: users.where(:teacher_prize_earned => true).count,
-    teacher_prizes_redeemed: teacher_prizes.where('user_id IS NOT NULL').count,
-    teacher_prizes_available: teacher_prizes.where('user_id IS NULL').count,
-
-    teacher_bonus_prizes_earned: users.where(:teacher_bonus_prize_earned => true).count,
-    teacher_bonus_prizes_redeemed: teacher_bonus_prizes.where('user_id IS NOT NULL').count,
-    teacher_bonus_prizes_available: teacher_bonus_prizes.where('user_id IS NULL').count,
+    accounts: {
+      users: all_users.count,
+      students: all_students.count,
+      teachers: all_teachers.count,
+    },
+    born_accounts: {
+      users: users.count,
+      students: students.count,
+      female_students: students.where(gender: 'f').count,
+      male_students: students.where(gender: 'm').count,
+      teachers: teachers.count,
+      female_teachers: teachers.where(gender: 'f').count,
+      male_teachers: teachers.where(gender: 'm').count,
+    },
+    actives: {
+      seven_day_students: seven_day_active_students.count,
+      seven_day_female_students: seven_day_active_students.where(gender: 'f').count,
+      seven_day_male_students: seven_day_active_students.where(gender: 'm').count,
+      thirty_day_students: thirty_day_active_students.count,
+      thirty_day_female_students: thirty_day_active_students.where(gender: 'f').count,
+      thirty_day_male_students: thirty_day_active_students.where(gender: 'm').count,
+    },
+    thirty_day_retention_historical: build_demographic_breakdowns(
+      thirty_day_retention_historical_baseline, thirty_day_retention_historical),
+    thirty_day_retention_current_weekly: build_demographic_breakdowns(
+      thirty_day_retention_current_weekly_baseline, thirty_day_retention_current_weekly),
+    thirty_day_retention_current_monthly: build_demographic_breakdowns(
+      thirty_day_retention_current_monthly_baseline, thirty_day_retention_current_monthly),
+    students_and_teachers: {
```

Comment:
Not at the moment.  Added a TODO(asher) to add additional student-teacher metrics and pipe them to the dashboard.